releases/v2.1.0-alpha.20180604.md

```diff
@@ -0,0 +1,214 @@
+---
+title: What&#39;s New in v2.1-alpha.20180604
+toc: false
+summary: Additions and changes in CockroachDB version v2.1-alpha.20180604 since version v2.1-alpha.20180507
+---
+
+## June 4, 2018
+
+This release includes general usability enhancements and bug fixes as well as one main benefit:
```

Comment:
I think we cut "one main benefit" -- and just say "as well as" -- since this is the highlighted benefit but not the main benefit (that would probably be usability).

Reply:
Done.